applications/nrf_desktop/configuration/nrf52833dk_nrf52833/app_ZDebug.conf

```diff
@@ -70,7 +70,6 @@ CONFIG_RESET_ON_FATAL_ERROR=n
 CONFIG_GPIO=y
 
 CONFIG_SERIAL=n
-CONFIG_UART_0_NRF_UART=n
```

Comment:
Is the UART disabled by default now in Zephyr?

Reply:
Please see the commit log which introduced this change.

Reply:
>nrf_desktop: adjust UART/SPI Kconfig instance settings

>The per-instance UART and SPI enable options are no longer
user-settable. They still exist (for now), but they obtain their
values directly from devicetree.

>Remove direct settings in favor of equivalent configuration via
devicetree overlay.

>In some cases, no devicetree changes are necessary. For example,
since UART_<x>_NRF_UART depends on UART_NRFX, which itself depends on
SERIAL, setting CONFIG_SERIAL=n is all that is needed to
ensure CONFIG_UART_<x>_NRF_UART is also n.

>In other cases, to be safe we ensure the corresponding devicetree node
is enabled. We also set its compatible to the appropriate value to
force the same Kconfig option value being removed.

>Leave UART_0, SPI_0 option settings as-is. Those are still
user-settable and even though they have no effect on the Nordic
drivers (which respect CONFIG_(UART|SPI)_(NUM)_NRF_(SUFFIX) instead),
some application code might be using them, so defer their removal
until they are removed from Zephyr itself.

>Signed-off-by: Martí Bolívar <marti.bolivar@nordicsemi.no>